docs/content/docs/guide/authoringprs.md

```diff
@@ -66,6 +66,41 @@ check out the code that is being tested.
 | trigger_comment     | The comment triggering the PipelineRun when using a [GitOps command]({{< relref "/docs/guide/running.md#gitops-command-on-pull-or-merge-request" >}}) (like `/test`, `/retest`) | `{{trigger_comment}}`               | /merge-pr branch                                                                                                                                              |
 | pull_request_labels | The labels of the pull request separated by a newline                                                                                                                           | `{{pull_request_labels}}`           | bugs\nenhancement                                                                                                                                             |
 
+### Defining Parameters with Object Values in YAML
+
+When working with YAML, especially in the context of defining params, you may want to pass an object in a dynamic variable (e.g., {{ body }})
+as the value of a parameter. However, YAML's validation rules do not allow this kind of value to be defined inline in flow scalars formats.
+
+When you attempt to define a parameter like this:
+
+```yaml
+spec:
+  params:
+    - name: body
+      value: {{ body }} # this way you will get yaml validation error
+  pipelineSpec:
+    tasks:
+```
+
+It results in a YAML validation error because the object or dynamic value cannot be placed directly in the flow scalar format (i.e., inline).
```

Comment:
you can just say inline, no need to say about flow scalare which is very internal to the yaml parser